src/parameterizations/vertical/MOM_geothermal.F90

```diff
@@ -136,6 +144,10 @@ subroutine geothermal(h, tv, dt, ea, eb, G, GV, CS, halo)
 !$OMP                                  wt_in_place,dTemp,dRcv,h_transfer,heating, &
 !$OMP                                  I_h)
 
+! Save temperature and thickness before any changes are made (for diagnostic)
+temp_old = tv%T
+h_old = h
```

Comment:
This block needs to be indented to align with the Fortran code below, and non-commented code above.
(The OMP block is a slightly different thing)